pkg/operation/etcdencryption/encryptionkeys_test.go

```diff
@@ -0,0 +1,94 @@
+package encryptionconfiguration
+
+import (
+	"crypto/rand"
+	b64 "encoding/base64"
+	"testing"
+	"time"
+)
+
+func TestEncryptionKeyCreation(t *testing.T) {
```

Comment:
same like above - please use the BDD style

Reply:
adapted tests